<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addServerPoke</class>
 <widget class="QDialog" name="addServerPoke">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>225</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>   虚拟训练师名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="list_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="sureBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>451</width>
     <height>117</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>    精灵</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="name_2">
          <item>
           <property name="text">
            <string>Bulbasaur</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Charmander</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Squirtle</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>    等级</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="level_2">
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>    风格</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="style_2">
          <item>
           <property name="text">
            <string>hpful</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>defendful</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>strong</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>fast</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>   攻击属性</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="power_2">
          <item>
           <property name="text">
            <string>excellent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>great</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bad</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>   生命属性</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="hp_2">
          <item>
           <property name="text">
            <string>excellent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>great</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bad</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>   防御属性</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="defense_2">
          <item>
           <property name="text">
            <string>excellent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>great</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bad</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>   敏捷属性</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="agile_2">
          <item>
           <property name="text">
            <string>excellent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>great</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bad</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
